### PR TITLE
OCPEDGE-996: IBU e2e serial conformance test

### PR DESCRIPTION
**Pull request by jeff-roche**

### CODE REVIEW

**Comment by jeff-roche** [comment]
periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.15-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.15-image-based-upgrade-e2e

**Comment by javipolo** [comment]
Is this also pinned to a ib-orchestrate-vm commit?

**Comment by jeff-roche** [comment]
> Is this also pinned to a ib-orchestrate-vm commit?

Good catch, I will add that

**Comment by jeff-roche** [comment]
@javipolo it is now pinned, I am not going to rerun the rehearse as it doesn't change the result

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.15-image-based-upgrade-e2e periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e

**Comment by jeff-roche** [comment]
The two e2e failures are acceptable failures (they are showing current regressions)

**Review by donpenney — APPROVED** [review]

File: ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main__nightly-4.15.yaml [hunk 1]
```diff
@@ -3,6 +3,10 @@ base_images:
     name: "4.15"
     namespace: ocp-kni
     tag: lifecycle-agent-operator
+  tests:
+    name: "4.16"
```
Comment (jc-rh) [inline ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main__nightly-4.15.yaml:7]: Is this the to version or from?

Comment (jeff-roche) [reply ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main__nightly-4.15.yaml:7]: neither, this is the version of the `openshift-tests` binary

**Pull request merged**